types/kakaomaps/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "dt-header": false,
+        "no-unnecessary-class": false
```

Comment:
It would be preferable to disable this error on each of the three lines where it occurs.

Reply:
Thanks, I fixed this issue as you recommended